node.js.sls

```diff
@@ -1,5 +1,5 @@
 node.js:
-  {% for version in ['7.0.0'] %}
+  {% for version in ['12.0.0', '11.14.0'] %}
```

Comment:
Please leave 7.0 for those who have it installed. Just add the newer versions. All versions still available here: https://nodejs.org/dist/